pronouncing/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 pronunciations = None
 lookup = None
+rhymeDict = None
```

Comment:
please follow pep8 conventions for variable names. would prefer `rhyme_lookup` or similar for this variable (no camelCase).